core/src/core/transaction.rs

```diff
@@ -1470,16 +1459,8 @@ impl Transaction {
 	}
 
 	/// Transaction minimum acceptable fee
-	pub fn accept_fee(&self, height: u64) -> u64 {
```

Comment:
This is a different rule change than that discussed in the RFC right?
The RFC is related to the 40 bit fee limit.
This specific change involves the minimum acceptable fee.

Or does the RFC also cover minimum `accept_fee` rules?

Reply:
The RFC covers all of 
* the 40-bit limit
*  the 4-bit fee_shift
* use of uniform weights
* new minimum accept fee base to make output cost about a grin cent